### DISCUSSION TITLE
How to replicate Wordpress shortcodes?

### DISCUSSION THREAD

**Discussion by jamiepickett** [datenstrom/community#620] [Ask a question] [answered]
Has anyone tried to replicate wordpress shortcodes in Yellow?  I have tried using the "shortcuts" but triggers containing dashes or underscores seem to not trigger a shortcut.

For example, I need to replace items like this within my content with a returned HTML block of code.

[code-create-card slug="/blogs/build-smart-on-kubernetes-developers-path-to-platform-freedom/" type="light" featured="true"]

**Comment by schulle4u** [comment, marked as answer]
The syntax of Yellow's shortcodes (or shortcuts as we call them) is a little different from WordPress. Instead of `key=value` pairs as parameter you only write the actual value, e. g. `[code-create-card "/blogs/build-smart-on-kubernetes-developers-path-to-platform-freedom/" light true]`. I suggest to have a look at the [helloworld extension](https://github.com/schulle4u/yellow-extension-helloworld) or any of the other official extensions, they all should provide a good starting point on how Yellow can be extended with new features.
